packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -481,9 +481,14 @@ export class RichText extends Component {
 			formatPlaceholder,
 		} );
 		this.lastEventCount = event.nativeEvent.eventCount;
-		// we don't want to refresh aztec as no content can have changed from this event
-		// let's update lastContent to prevent that in shouldComponentUpdate
-		this.lastContent = this.removeRootTagsProduceByAztec( unescapeSpaces( text ) );
+		
+		// Make sure there are changes made to the content before upgrading it upward
+		if ( this.lastContent !== this.removeRootTagsProduceByAztec( unescapeSpaces( text ) ) ) {
```

Comment:
Not a blocker, but I wonder if we can extract `this.removeRootTagsProduceByAztec( unescapeSpaces( text )` into a local variable to DRY this up a little. Maybe something like `innerContent`? WDYT?

Reply:
Looks good now. :+1: